psydac/fem/partitioning.py

```diff
@@ -280,21 +300,59 @@ def construct_reduced_interface_spaces(spaces, reduced_spaces, interiors, connec
                     cart_ij.set_interface_communication_infos(get_minus_starts_ends, get_plus_starts_ends)
                     Vi.create_interface_space(axis_i, ext_i, cart=cart_ij)
                     Vj.create_interface_space(axis_j, ext_j, cart=cart_ij)
+
+                    cart = InterfaceCartDecomposition(cart_ij._cart_minus, cart_ij._cart_plus,
+                                                      cart_ij._comm, [axis_i, axis_j], [ext_i, ext_j],
+                                                      [cart_ij.ranks_in_topo_minus, cart_ij.ranks_in_topo_plus],
+                                                      [cart_ij._local_group_minus, cart_ij._local_group_plus],
+                                                      [cart_ij._local_comm_minus, cart_ij._local_comm_plus],
+                                                      [cart_ij._root_rank_minus, cart_ij._root_rank_plus],
+                                                      [], reduce_elements=True)
+
+                    cart.set_interface_communication_infos(get_minus_starts_ends, get_plus_starts_ends)
+                    max_ncells = tuple(max(ni,nj) for ni,nj in zip(Vi.ncells,Vj.ncells))
+                    Vi._refined_space[max_ncells].create_interface_space(axis_i, ext_i, cart=cart)
+                    Vj._refined_space[max_ncells].create_interface_space(axis_j, ext_j, cart=cart)
```

Comment:
See above

Reply:
done in [f74fd12](https://github.com/pyccel/psydac/pull/241/commits/f74fd122c1b075e8f616d556a9b44aa8408bb4a1)